Polyfills/URL/Source/URL.cpp

```diff
@@ -92,32 +86,27 @@ namespace Babylon::Polyfills::Internal
         return m_searchParamsReference.Value();
     }
 
-    //TODO replace stub with actual implementation
-    Napi::Value URL::CreateObjectURL(const Napi::CallbackInfo& info)
-    {
-        return Napi::Value::From(info.Env(), "");
-    }
-
     URL::URL(const Napi::CallbackInfo& info)
         : Napi::ObjectWrap<URL>{info}
         , m_runtimeScheduler{JsRuntime::GetFromJavaScript(info.Env())}
     {
-        if (!info.Length())
+        if (!info.Length()) {
```

Comment:
nit: bunch of places
```suggestion
        if (!info.Length())
        {
```